gitgud/skills/rework/__init__.py

```diff
@@ -0,0 +1,12 @@
+import pkg_resources
+
+from gitgud.skills.util import BasicLevel
+from gitgud.skills.util import Skill
+
+skill = Skill(
+    'rework',
+    [
+        BasicLevel('cherrypick', pkg_resources.resource_filename(__name__, '_cherrypick/')),
```

Comment:
You shouldn't have the cherrypicking level added here yet. We'll add them in one as a time automatically as we merge